### PR TITLE
[Code for Equity Fellowship] updates

### PR DESCRIPTION
**Pull request by nikhilro**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -23,4 +23,4 @@ url: 'https://impactlabs.io/'
 twitter:
   username: impactlabs_io
 
-logo: 'https://impactlabs.io/assets/images/social_media_banner.jpg'
+logo: 'https://impactlabs.io/assets/images/impact_labs.jpg'
```
Comment (nikhilro) [inline _config.yml:26]: couldn't find social_media_banner.jpg

Comment (Holben888) [reply _config.yml:26]: Yeah this will work. To be honest, I think we should delete this line. The SEO plugin doesn't do anything with it

**Review by Holben888 — APPROVED** [review]

File: _config.yml (hunk 1 above)
Comment (nikhilro) [reply _config.yml]:
```
logo - URL to a site-wide logo (e.g., /assets/your-company-logo.png) - If you would like the "publisher" property to be present, you must add this field to your site's configuration, during the validation of the structured data by Google Search Console, if the logo field is not validated, you will find errors inherent to the publisher in the Rich Results Testing Tool
```

the documentation seems to suggest something?

**Comment by nikhilro** [comment]
are you sure it's unused?

**Comment by Holben888** [comment]
> are you sure it's unused?

Yep! Checked the before and after in the `head`. Doesn't get used for anything; our favicon is a separate file

**Comment by nikhilro** [comment]
popped the last commit, merging now

**Pull request merged**